doc/usage.rst

```diff
@@ -52,9 +52,14 @@ which are separated by newlines. The following keys are defined:
     * if set to ``True``, when ``run_flow_on_task`` or similar methods are called a lookup is performed to see if there already exists such a run on the server. If so, download those results instead.
     * if not given, will default to ``True``.
 
+* retry_policy:
+    * Defines how to react when the server is unavailable or experiencing high load. It determines both how often to attempt to reconnect and how quickly to do so. Please don't use ``human`` in an automated script that you run more than one instance of, it might increase the time to complete your jobs and that of others.
+    * human (default): For people running openml in interactive fashion. Try only a few times, but in quick succession.
+    * robot: For people using openml in an automated fashion. Keep trying to reconnect for a longer time, quickly increasing the time between retries.
+
 * connection_n_retries:
-    * number of connection retries.
-    * default: 2. Maximum number of retries: 20.
+    * number of connection retries
+    * default depends on retry_policy (5 for ``human``, 50 for ``robot``)
```

Comment:
Don't we need to set the retry policy in the file `testing.py` to enable policy `robot` for the unit tests? Also, is the only difference the number of re-tries?

Reply:
Sorry, for the confusion (I don't think I pinged you for review yet?).
I paused so I could wait and process the discussion of #1072 and merge of #1071.

Reply:
> Sorry, for the confusion (I don't think I pinged you for review yet?).

Sorry, I'm sometimes to eager ;)